utils/tessdata/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2019 Valentin Kivachuk
```

Comment:
Add email. Similar to this: https://github.com/openwrt/packages/blob/master/admin/ipmitool/Makefile#L2